ui/admin/app/components/form/target/details/index.hbs

```diff
@@ -325,6 +325,42 @@
     </Hds::Form::TextInput::Field>
   {{/if}}
 
+  {{#if
+    (and @model.isNew (can 'create model' @globalScope collection='aliases'))
+  }}
+
+    <Form::Field::ListWrapper
+      @layout='horizontal'
+      @isOptional={{true}}
+      @disabled={{form.disabled}}
+    >
+      <:fieldset as |F|>
+        <F.Legend>
+          {{t 'resources.alias.form.alias.label'}}
+        </F.Legend>
+        <F.HelperText>
+          {{t 'resources.alias.form.alias.help'}}
+        </F.HelperText>
+
+        {{#if @model.errors.with_aliases}}
+          <F.Error as |E|>
+            {{#each @model.errors.with_aliases as |error|}}
+              <E.Message>{{error.message}}</E.Message>
+            {{/each}}
+          </F.Error>
+        {{/if}}
+      </:fieldset>
+
+      <:field as |F|>
```

Comment:
None blocking comment: I am seeing when adding aliases to a target, every input has the same `aria-label=Value`. I know this is not specific to this PR and more to the component that we use, but what do you think? should we try to get a more descriptive `aria-label` or value is good enough?

Reply:
Good point- I agree, we should use a descriptive label. I'll think more about this and do a refactor separately to update the component